### PR TITLE
created models

### PR DESCRIPTION
**Pull request by Big-Hdub**

### CODE REVIEW

File: instance/dev.db
Comment (Forestbriggs) [inline instance/dev.db:1]: dev.db should not be pushed to github. Add to gitignore and recommit without it

File: Pipfile [hunk 1]
```diff
@@ -25,6 +25,8 @@ markupsafe = "==2.1.2"
 sqlalchemy = "==1.4.46"
 werkzeug = "==2.2.2"
 wtforms = "==3.0.1"
+importlib-metadata = "==6.9.0"
```
Comment (Forestbriggs) [inline Pipfile:28]: Where are these 2 new packages being used? Were they installed by mistake?
(referencing importlib-metadata and zipp)

File: app/models/comment.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, nullable=False)
+    type_id = db.Column(db.Integer, nullable=False)
+    _type = db.Column('type', db.String(20), nullable=False)
+    comment = db.Column(db.String(255), nullable=False)
+    created_at = db.Column(db.DateTime, default=func.now())
+    updated_at = db.Column(db.DateTime, default=func.now())
+
+    def __init__(self, user_id, type_id, type, comment):
```
Comment (Forestbriggs) [inline app/models/comment.py:19]: Is there a reason for using the __init__ on some of these models? (goes for multiple model files)

File: app/models/up_down_vote.py
Comment (Forestbriggs) [inline app/models/up_down_vote.py:1]: Model names should be camel cased

File: migrations/versions/20240622_155509_.py
Comment (Forestbriggs) [inline migrations/versions/20240622_155509_.py:1]: Recreate migration with updated models

**Review by Forestbriggs — CHANGES_REQUESTED** [review]
A few questions/changes to be made. Overall, it looks good just a few discrepancies.

File: app/models/comment.py (hunk 2 above)
Comment (Big-Hdub) [reply app/models/comment.py:19]: Redefining the _init_ to do the validation.

File: Pipfile (hunk 1 above)
Comment (Big-Hdub) [reply Pipfile:28]: Actually not sure how those are there.

File: app/models/up_down_vote.py
Comment (Big-Hdub) [reply app/models/up_down_vote.py:1]: Oops did a mix of camel and snake case in the name lol

File: Pipfile (hunk 1 above)
Comment (Sohna-AI) [reply Pipfile:28]: Both import were included in the starter files for the project

**Pull request merged**